script/get-exports.mjs

```diff
@@ -0,0 +1,453 @@
+import {existsSync} from 'node:fs'
```

Comment:
Sorry, I'm a bit confused, where would we like to call this script from? after `./script/build` (npm run build)?

Reply:
@siddharthkp whoops! This was my test script, removed in the latest commit since it's in the test file itself 👍